src/components/Card/Card.js

```diff
@@ -1,30 +1,81 @@
 import { Tile } from "carbon-components-react";
 import React, { useState } from "react";
+import CardContent from "./CardContent/CardContent";
+import CardControls from "./CardControls/CardControls";
+import CardHeader from "./CardHeader/CardHeader";
+
+let currentValues = {
+  header: "Header",
+  content: "Content",
+};
 
 const Card = () => {
-  const [isChecked, setIsChecked] = useState(false);
+  const [state, setState] = useState({
+    isChecked: false,
+    isEditMode: false,
+    headerValue: currentValues.header,
+    contentValue: currentValues.content,
```

Comment:
better store the whole object and pass the whole object data. Otherwise if any property added - you'll have to put changes everywhere in the code to pass the added data.